MoviesExample/app/src/main/res/layout/movies_list_card.xml

```diff
@@ -15,6 +22,8 @@
     layout="@layout/movies_list_item"
     android:layout_margin="16dp"
     android:layout_width="match_parent"
-    android:layout_height="match_parent" />
+    android:layout_height="match_parent"
+    app:movie="@{movie}"/>
```

Comment:
This is interesting as well. As you can see this is an `include` layout. With `app:movie` we are passing the value of `{@movie}` to `movies_list_item`

Reply:
this is great! so you don't have to define a binder for this 👍